apps/demo/translations/src/en/validation.json

```diff
@@ -46,5 +46,14 @@
   },
   "appealStages": {
     "required": "Select an appeal stage from the list"
+  },
+  "saveForm": {
+    "required": "Select an option below"
+  },
+  "saveEmail": {
+    "required": "Enter your email address"
+  },
+  "saveRef": {
+    "required": "Create a memorable reference"
```

Comment:
You must enter a reference

Reply:
changed line 57 to 'You must enter a reference'